### PR TITLE
[nomergey] Add live transaction view

### PR DESCRIPTION
**Pull request by nathanleclaire**
Makes much more sense to follow along with processed transactions than with program/account changes. 

So this is the start of a port to do that in Live View instead. This also will be the foundation for putting https://github.com/workbenchapp/solana-workbench/pull/70 to good use.

### CODE REVIEW

**Comment by nathanleclaire** [comment]
still lots to do parsing the data but getting there 

Screen Shot 2022-04-05 at 1 22 31 PM

File: src/renderer/components/TransactionView.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  const { net } = props;
+  const [latestProcessedSlot, setLatestProcessedSlot] = useState(0);
+  const [renderedParsedTransactions, setRenderedParsedTransactions] = useState<
+    (sol.ParsedTransactionWithMeta | null)[][]
+  >([]);
+  useEffect(() => {
+    const blockInterval = setInterval(async () => {
+      const solConn = new sol.Connection(netToURL(net));
+      const blockProduction = await solConn.getBlockProduction();
+      const blocks = await solConn.getBlocks(
```
Comment (SvenDowideit) [inline src/renderer/components/TransactionView.tsx:24]: I was going to ask for the subscription code to be separated out into `../data/transactions.ts`, but actually - this feels small and targeted, and who needs premature extraction :)

**Comment by SvenDowideit** [comment]
I get `Uncaught (in promise) TypeError: solConn.getBlockProduction is not a function    at TransactionView.tsx:23`

and that gets fixed by using a more recent version of `solana/web3.js` - i went to 1.37.1 - https://www.npmjs.com/package/@solana/web3.js

but then running it, I get a frozen electron app, so I guess I'll wait (tbh, I think I'll add this view as the bottom section on my pr, and see what having both program changes and transactions gives me :)

**Comment by nathanleclaire** [comment]
Yea this isn't near ready for prime time yet, added `[nomergey]`

**Comment by nathanleclaire** [comment]
This is getting pretty close for review, remaining items:

- [ ] fix issue w/ thrown exception when can't connect to validator
- [ ] get working w/ higher nets (load up only one block instead of trying to track all, maybe with refresh button?)
- [ ] not sure i unfurl/bundle/render the signatures and account data correctly, I think that `accountInstructionData` map is wrong and needs to be fixed 

image

**Comment by nathanleclaire** [comment]
badly wanna get some TXN integration in there but, this PR is basically toast at this point -- we'll have to do some pondering on how we want to build out something like this (follow every txn was a bit much)

**Pull request closed without merging**